src/utils/useDimensions.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react';
```

Comment:
We need to use star import here, otherwise, it will break when imported in a project through npm

```suggestion
import * as React from 'react';
```